smslibrary/src/main/java/com/eis/smslibrary/SMSMessageHandler.java

```diff
@@ -71,7 +72,7 @@ public String parseData(@NonNull final SMSMessage message) {
 
     public class DefaultSMSMessageParseStrategy implements MessageParseStrategy<String, SMSPeer, SMSMessage> {
 
-        static final String HIDDEN_CHARACTER = "\r";
+        static final String SIGNATURE = " ";
```

Comment:
I would return to the original name `HIDDEN_CHARACTER`, because it's mentioned a few times in the documentation of other files. Either this or edit the documentation of those files.